### ISSUE TITLE
VB.NET Support dropped in WebAPI (vNext) also!?

### ISSUE THREAD

**Issue by aarondglover** [aspnet/Home#446]
Now that the ASP.NET vNext Teams and WebAPI projects are merged, can someone please confirm for me some information about VB.NET

There have been recent announcements/confirmation that VB.NET **WILL NOT** be supported in ASP vNext (a TERRIBLE decision IMHO which I won't talk about here) . 

With the merging of the WebAPI and ASP.NET vNext projects does that mean that VB.NET developers not only lose the ability to develop ASP.NET MVC applications BUT ALSO WEBAPI projects (in the new version of course) ?

It seems to be implied that with ASP.NET vNext dropping support for VB.NET, that this will also extend to WebAPI also but please provide clarification on this.

**Simply put; Will WebAPI vNext support VB.NET?**

**Comment by GuardRex** [comment]
Unfortunately, read this:

https://github.com/aspnet/Home/issues/349

**Comment by aarondglover** [comment]
GuardRex; yes I have found that issue thankyou. Perhaps someone can however answer clearly and succinctly;

**Will WebAPI vNext support VB.NET?**

I think a YES, NO answer is more than adequate. I want to remove any ambiguity surrounding this issue given the "merging" of the two projects (WebAPI & ASP.NET vNext).

**Comment by Eilon** [comment]
Web API and MVC were merged into a single project called ASP.NET MVC 6, which is part of ASP.NET 5. As such, there are no current plans to directly support VB in this project.

You can read more in the discussion on this issue: https://github.com/aspnet/Home/issues/349

**Comment by aarondglover** [comment]
The argument or justification for dropping support of VB.NET has largely been that there is very little usage of these by VB.NET developers; and this is the reason they are dropping support of VB.NET.

That is a self fulfilling prophecy as documentation and samples in VB.NET are rarely provided. How to do they expect early adoption of these new framework features when they cause greatly more friction for a VB.NET developer to get started with them.

**Comment by Eilon** [comment]
Hey everyone, good news! We've announced plans to support VB in ASP.NET 5 / DNX!

Read more on the official blog post: http://blogs.msdn.com/b/webdev/archive/2015/04/23/making-it-better-asp-net-with-visual-basic-14.aspx

Stay tuned for updates.
